Tasks/Npm/task.json

```diff
@@ -9,7 +9,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 3
+        "Patch": 13
```

Comment:
This needs to be 1.0.4. Build the task and include in the PR all the files changed within the task by the build.